python/test/test_exception.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+
+# Copyright (c) 2018, MariaDB Corporation. All rights reserved.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301  USA
+
+import pytest
+import pymcsapi
+
+def test_exception_thrown():
+    with pytest.raises(Exception):
```

Comment:
Shouldn't there be  a with pytest.raises for each call below?

Reply:
According to the documentation "with pytest.raises(Exception):" fails if Exception isn't raised in the whole block. Refined the test to exactly check if the table was not existent.